src/components/search/search.jsx

```diff
@@ -75,6 +76,29 @@ const SearchBox = forwardRef(
 			dismiss,
 		] );
 
+		useEffect( () => {
+			const handleKeyDown = ( event ) => {
+				if ( event.key === '/' ) {
```

Comment:
Checking only the `/` key might cause some issues. Instead, we can combine `meta key`+ `/`.

Reply:
Resolved.